script_runner/tests/test_ctx_wrapper.py

```diff
@@ -142,7 +142,8 @@ def test_logger(self, capture):
             'error_message'
         ]
         self._run(script)
-        # first message is unrelated to the test
+        # first two messages are unrelated to the test
+        capture.records.pop(0)
```

Comment:
I think that `del capture.records[:2]` or `capture.records = capture.records[2:]` is more intuitive..